include/swift/Serialization/ModuleFormat.h

```diff
@@ -851,7 +851,7 @@ namespace decls_block {
     FunctionTypeRepresentationField, // representation
     BCFixed<1>,          // throws?
     // SWIFT_ENABLE_TENSORFLOW
-    BCFixed<1>,          // differentiable?
+    BCFixed<2>,          // differentiable & linear?
```

Comment:
For Generic Function Type, this and also [this line](https://github.com/apple/swift/pull/25325/files#diff-5fa18b91a026a0b29b5c3b4d59feaf95R4728) were the only thing I updated. I'm not sure whether I need to update anymore.

Reply:
The comment should be changed to `differentiability kind`.